.github/workflows/docker_publish.yml

```
@@ -255,35 +264,45 @@ jobs:
           cd $GITHUB_WORKSPACE/tests
           ./travis-run-tests.sh python3
 
+  # if the previous step that tests the docker images passes then the images
+  # can be copied from the ghcr where they are saved using :ci_testing tags to
+  # :latest and :stable tags.
   pushing_test_stable_img:
-    if: ${{ github.repository_owner == 'pyne' }}
+    # if: ${{ github.repository_owner == 'pyne' }}
+    if: ${{ github.repository_owner == 'shimwell' }}
```

This just needs to be changed back, this was a temporary change made so that I could test the entire ci action

```suggestion
    if: ${{ github.repository_owner == 'pyne' }}
```